client/post-editor/copy-post/accordion.jsx

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
It's a bit inconsistent in our current codebase, but more recently I've been placing accordion files within the `post-editor/editor-drawer` directory with minimal logic necessary to render the accordion, deferring the bulk of the display to the individual `post-editor` component. For example, [`<EditorDrawerPageOptions />`](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-drawer/page-options.jsx) is the accordion component to render [`<EditorPageParent />`](https://github.com/Automattic/wp-calypso/tree/master/client/post-editor/editor-page-parent), [`<EditorPageTemplates />`](https://github.com/Automattic/wp-calypso/tree/master/client/post-editor/editor-page-templates), and [`<EditorPageOrder />`](https://github.com/Automattic/wp-calypso/tree/master/client/post-editor/editor-page-order). Keeping them separate better reflects that those individual components don't necessarily need to be rendered in the context of an accordion and allows us to separate distinct functionality (in the above case, where would the `accordion.jsx` file be otherwise located?)